CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.10)
+
+project(greentea-client
+    VERSION 0.1
+    DESCRIPTION "Greentea client"
+    LANGUAGES C CXX
+)
+
+# Note: The INTERFACE type is currently required by Mbed OS
```

Comment:
What's this about INTERFACE? I don't see INTERFACE.

Reply:
Good catch, removed now